docs/README.md

```diff
@@ -1,121 +1,240 @@
+```
+</div>
+<div id="YCSB_1" class="tabcontent" markdown="1">
+
+To create tables for YCSB benchmarking ([`ycsb-schema.json`](https://github.com/scalar-labs/scalardb-benchmarks/blob/master/ycsb-schema.json)), run the following command, replacing the contents in the angle brackets as described:
 
-For example, execute the following command with `tpcc-schema.json` to create tables for TPC-C benchmark.
-For YCSB and multi-storage YCSB, use `ycsb-schema.json` and `ycsb-multi-storage-schema.json`, respectively.
+```console
+$ java -jar scalardb-schema-loader-<VERSION>.jar --config /<PATH_TO>/scalardb.properties -f ycsb-schema.json --coordinator
+```
+</div>
+<div id="multi-storage_YCSB_1" class="tabcontent" markdown="1">
+
+To create tables for multi-storage YCSB benchmarking ([`ycsb-multi-storage-schema.json`](https://github.com/scalar-labs/scalardb-benchmarks/blob/master/ycsb-multi-storage-schema.json)), run the following command, replacing the contents in the angle brackets as described:
 
 ```console
-java -jar scalardb-schema-loader-<version>.jar --config /path/to/scalardb.properties -f tpcc-schema.json --coordinator
+$ java -jar scalardb-schema-loader-<VERSION>.jar --config /<PATH_TO>/scalardb.properties -f ycsb-multi-storage-schema.json --coordinator
 ```
+</div>
+</div>
+
+### Prepare a benchmarking configuration file
+
+To run a benchmark, you must first prepare a benchmarking configuration file. The configuration file requires at least the locations of the workload modules to run and the database configuration. 
+
+The following is an example configuration for running the TPC-C benchmark. The configurations under `database_config` should match the [benchmarking environment that you previously set up](#set-up-your-environment).
+
+{% capture notice--info %}
+**Note**
+
+Alternatively, instead of specifying each database configuration item in the `.toml` file, you can use the ScalarDB properties file. If `config_file` is specified (commented out below), all other configurations under `database_config` will be ignored.
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+```toml
+[modules]
+[modules.preprocessor]
+name = "com.scalar.db.benchmarks.tpcc.TpccLoader"
+path = "./build/libs/scalardb-benchmarks-all.jar"
+[modules.processor]
+name = "com.scalar.db.benchmarks.tpcc.TpccBench"
+path = "./build/libs/scalardb-benchmarks-all.jar"
+[modules.postprocessor]
+name = "com.scalar.db.benchmarks.tpcc.TpccReporter"
+path = "./build/libs/scalardb-benchmarks-all.jar"
+
+[database_config]
+contact_points = "localhost"
+contact_port = 9042
+username = "cassandra"
+password = "cassandra"
+storage = "cassandra"
+#config_file = "/<PATH_TO>/scalardb.properties"
+```
+
+You can define static parameters to pass to modules in the configuration file. For details, see the sample configuration files below and available parameters in [Common parameters](#common-parameters):
```

Comment:
> You can define static parameters

I wonder what this "`static` parameters" means. It would be great if we make it clearer 🙇

Reply:
I don't remember well, but I guess I just followed the [Kelpie document](https://github.com/scalar-labs/kelpie). I think we can say just `parameters` here.